sensor/common/detector/metrics/metrics.go

```diff
@@ -180,6 +180,27 @@ var (
 		Name:      "detector_network_flows_dropped_total",
 		Help:      "A counter of the total number of network flows that were dropped if the detector buffer was full",
 	})
+	detectorBlockScanCalls = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "block_scan_calls",
```

Comment:
Following https://prometheus.io/docs/practices/naming/
```suggestion
		Name:      "block_scan_calls_total",
```

Reply:
Done